index.html

```diff
@@ -115,19 +134,29 @@ <h3>Multi-Post Stories</h3>
                 <li class="padding-right">Ruby</li>
             </ul>
             <button type="button" class="project-button">See Project</button>
+
+            </div>            
+        
         </section> 
     </section>
     <section class="form-container">
-        <h3>CONTACT US</h3>
+        <h3 class="mobH">CONTACT US</h3>
```

Comment:
- [ ] Kindly adjust the spacing between the projects and contact section, as specified in the design.